### DISCUSSION TITLE
setup remote sensor - using openVpn

### DISCUSSION THREAD

**Discussion by fadoh** [Security-Onion-Solutions/securityonion#2683] [Unsupported Versions] [answered]
Hi, 

Install source. perform a network installation on Ubuntu 18.04
Install type. forward node,
connecting to the manager using OpenVPN client 
 Installation fails at the beginning stage, with the following message:
The IP being routed by Linux is not the IP address assigned to the management interface  (tun0)
this is not a supported configuration  please remediate and rerun

**Comment by TOoSmOotH** [comment, marked as answer]
Currently installation over VPN is not supported at the moment. #1139 is tracking this.

**Comment by fadoh** [comment]
Thank you for your response

Do we have any other work a round . If I want to setup the sensor in remote
location .

Like having site to site vpn .

or I have to setup data node in remote location .


Regards
